### ISSUE TITLE
Freeform migration: Cannot assign string to property verbb\formie\fields\formfields\Agree::$description of type ?array

### ISSUE THREAD

**Issue by LauraStatik** [verbb/formie#1433]
### Describe the bug

We are trying to migrate our forms from Freeform to Formie but get an error on the Checkbox-fields.
 ```
Exception: Cannot assign string to property verbb\formie\fields\formfields\Agree::$description of type ?array (/Users/statik/Sites/intcra/vendor/verbb/formie/src/migrations/MigrateFreeform.php:550)
#0 /Users/statik/Sites/intcra/vendor/verbb/formie/src/migrations/MigrateFreeform.php(426): verbb\formie\migrations\MigrateFreeform->_mapField(Object(Solspace\Freeform\Fields\CheckboxField))
#1 /Users/statik/Sites/intcra/vendor/verbb/formie/src/migrations/MigrateFreeform.php(131): verbb\formie\migrations\MigrateFreeform->_buildFieldLayout(Object(Solspace\Freeform\Models\FormModel))
#2 /Users/statik/Sites/intcra/vendor/verbb/formie/src/migrations/MigrateFreeform.php(77): verbb\formie\migrations\MigrateFreeform->_migrateForm()
#3 /Users/statik/Sites/intcra/vendor/craftcms/cms/src/db/Migration.php(49): verbb\formie\migrations\MigrateFreeform->safeUp()
#4 /Users/statik/Sites/intcra/vendor/verbb/formie/src/controllers/MigrationsController.php(103): craft\db\Migration->up()
#5 [internal function]: verbb\formie\controllers\MigrationsController->actionFreeform()
#6 /Users/statik/Sites/intcra/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#7 /Users/statik/Sites/intcra/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#8 /Users/statik/Sites/intcra/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('freeform', Array)
#9 /Users/statik/Sites/intcra/vendor/craftcms/cms/src/web/Application.php(301): yii\base\Module->runAction('formie/migratio...', Array)
#10 /Users/statik/Sites/intcra/vendor/craftcms/cms/src/web/Application.php(625): craft\web\Application->runAction('formie/migratio...', Array)
#11 /Users/statik/Sites/intcra/vendor/craftcms/cms/src/web/Application.php(280): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#12 /Users/statik/Sites/intcra/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#13 /Users/statik/Sites/intcra/public/index.php(13): yii\base\Application->run()
#14 {main}
```

Are we doing something wrong?

### Steps to reproduce

1.  Install Freeform and create a form with a checkbox
2.  Open the migration tool from Formie to migrate this form

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.3.2

### Plugin version

2.0.28

### Multi-site?

Yes

### Additional context

Freeform plugin version: 4.0.7

**Comment by Gerbenodk** [comment]
I have the same error; same versions

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.28`

**Comment by engram-design** [comment]
Fixed in [2.0.29](https://github.com/verbb/formie/releases/tag/2.0.29)
